### PR TITLE
Adding a color validator and updated the cfp colors

### PR DESCRIPTION
**Pull request by jespino**

### CODE REVIEW

File: tools/validateCfpColor.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const fs = require("fs");
+
+const allEvents = JSON.parse(fs.readFileSync('../page/src/misc/all-events.json'), 'utf-8');
+
+for (const event of allEvents) {
+    if (event.cfp && event.cfp.untilDate && new Date(event.cfp.untilDate) < new Date() && event.misc.indexOf("color=green") !== -1) {
+        console.log("Invalid cfp green color: " + new Date(event.date[0]).toISOString() + " " + event.name);
```
Comment (scraly) [inline tools/validateCfpColor.js:7]:
```suggestion
        console.log("Invalid CFP green color: " + new Date(event.date[0]).toISOString() + " " + event.name);
```

File: tools/validateCfpColor.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    }
+    if (event.cfp && event.cfp.untilDate && new Date(event.cfp.untilDate) >= new Date() && event.misc.indexOf("color=red") !== -1) {
+        console.log("Invalid cfp red color: " + new Date(event.date[0]).toISOString() + " " + event.name);
```
Comment (scraly) [inline tools/validateCfpColor.js:10]:
```suggestion
        console.log("Invalid CFP red color: " + new Date(event.date[0]).toISOString() + " " + event.name);
```

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks, it's a good idea :) 

Please edit `run.sh` script also with a new line at the end:

```
node validateCFPColor.js
```

And change `validateCfpColor.js` to `validateCFPColor.js`

**Comment by jespino** [comment]
Done

**Comment by scraly** [comment]
gracias! :)

**Pull request merged**